doc/source/user_guide/computation.rst

```diff
@@ -229,6 +229,13 @@ see the :ref:`groupby docs <groupby.transform.window_resample>`.
 
    The API for window statistics is quite similar to the way one works with ``GroupBy`` objects, see the documentation :ref:`here <groupby>`.
 
+.. warning::
+
+    When using ``rolling()`` and an associated function the results are calculated with rolling sums. As a consequence
+    when having values differing with magnitude :math:`1/np.finfo(np.double).eps` this results in truncation. It must be
```

Comment:
also might be worth noting that we use kahan summation (add a wiki leak) to preserve accuracy as much as possible)

Reply:
Done